crates/polars-plan/src/utils.rs

```diff
@@ -192,11 +192,12 @@ pub(crate) fn get_single_leaf(expr: &Expr) -> PolarsResult<Arc<str>> {
             Expr::SortBy { expr, .. } => return get_single_leaf(expr),
             Expr::Window { function, .. } => return get_single_leaf(function),
             Expr::Column(name) => return Ok(name.clone()),
+            Expr::Count => return Ok(Arc::from(COUNT)),
```

Comment:
IIUC, `Expr::Count` is a leaf expr, It is different from `AggExpr` which does have an input.

Reply:
Good catch!